CMakeLists.txt

```diff
@@ -112,9 +112,7 @@ if(PYTHON_BINDINGS)
       set(PYTHON_LIBRARIES "$ENV{PYTHON_LIBRARIES}")
     endif()
     if(NOT PYTHON_INCLUDE_DIRS AND NOT PYTHON_LIBRARIES)
-        find_package(PythonLibs 2.7 REQUIRED)
-      else()
-        message(FATAL_ERROR "Inconsistent PYTHON_INCLUDE_DIRS and PYTHON_LIBRARIES")
```

Comment:
D:\WorkData\Instrumentation\Triton\build>cmake -G "Visual Studio 14 2015 Win64" -DPYTHON_INCLUDE_DIRS="C:/Python27/include" -DPYTHON_LIBRARIES="C:/Python27/libs/python27.lib" -DBOOST_ROOT="D:/OpenSrc/boost_1_63_0" -DZ3_INCLUDE_DIRS="D:/OpenSrc/z3-4.5.0-x64-win/include" -DZ3_LIBRARIES="D:/OpenSrc/z3-4.5.0-x64-win/bin/libz3.lib" -DCAPSTONE_INCLUDE_DIRS="D:/OpenSrc/capstone-3.0.5-rc2-win64/include" -DCAPSTONE_LIBRARIES="D:/OpenSrc/capstone-3.0.5-rc2-win64/capstone.lib" ..
CMake Error at CMakeLists.txt:117 (message):
  Inconsistent PYTHON_INCLUDE_DIRS and PYTHON_LIBRARIES


-- Configuring incomplete, errors occurred!
See also "D:/WorkData/Instrumentation/Triton/build/CMakeFiles/CMakeOutput.log".

D:\WorkData\Instrumentation\Triton\build>